src/wasm-interpreter.h

```diff
@@ -240,7 +240,7 @@ class ExpressionRunner : public Visitor<SubType, Flow> {
         case ConvertUInt32ToFloat64: return value.convertUToF64();
         case ConvertSInt32ToFloat32: return value.convertSToF32();
         case ConvertSInt32ToFloat64: return value.convertSToF64();
-        default: abort();
+        default: WASM_UNREACHABLE();
```

Comment:
The latter, yeah. This is in the interpreter, and everything it receives must be valid code, so it's an internal compiler error if it isn't.